build/commands/lib/util.js

```diff
@@ -628,7 +628,9 @@ const util = {
       real_rewrapper: path.join(config.realRewrapperDir, 'rewrapper'),
     }
 
-    util.runGnGen(config.nativeRedirectCCDir, buildArgs)
+    util.runGnGen(config.nativeRedirectCCDir, buildArgs, [
+      '--root-target=//brave/tools/redirect_cc'
```

Comment:
I just found this flag recently. There's a list of "global flags" right at the bottom of the GN reference page: https://gn.googlesource.com/gn/+/master/docs/reference.md#switch_list